runtime/jvmti/jvmtiHook.c

```diff
@@ -505,6 +509,56 @@ jvmtiHookVirtualThreadUnmount(J9HookInterface **hook, UDATA eventNum, void *even
 
 #endif /* JAVA_SPEC_VERSION >= 19 */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void
+jvmtiHookVMCheckpoint(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
+{
+	J9CheckpointEvent * data = eventData;
+	J9JVMTIEnv * j9env = userData;
+	jvmtiExtensionEvent callback = *J9JVMTI_EXTENSION_CALLBACK(j9env, J9JVMTI_EVENT_OPENJ9_VM_CHECKPOINT);
+
+	Trc_JVMTI_jvmtiHookVMCheckpoint_Entry();
+
+	if (NULL != callback) {
+		J9VMThread * currentThread = data->currentThread;
```

Comment:
The same for `currentThread` as https://github.com/eclipse-openj9/openj9/pull/17252/files#r1176818684.